crates/dojo-core/src/integer.cairo

```diff
@@ -56,12 +56,30 @@ impl ContractAddressIntoU250 of Into<ContractAddress, u250> {
     }
 }
 
+impl U8IntoU250 of Into<u8, u250> {
+    fn into(self: u8) -> u250 {
+        u250 { inner: self.into() }
+    }
+}
```

Comment:
I think we can have a single generic impl like:

```suggestion
impl TIntoU250<T, impl TInto: Into<T, felt252>> of Into<T, u250> {
    fn into(self: T) -> u250 {
        let inner = TInto::into(self) ;
        // TODO: Bounds checks
        u250 { inner }
    }
}
```

Reply:
ooooh that's very cool indeed, well done

Reply:
im stuck on a this error:
```
error: Plugin diagnostic: Trait `core::traits::Into::<dojo_core::integer::u250, dojo_core::integer::u250>` has multiple implementations, in: "dojo_core::integer::TIntoU250", "core::traits::TIntoT"
   --> RouteAuth:38:180
                          IWorldDispatcher { contract_address: world_address }.set_entity(dojo_core::string::ShortStringTrait::new('AuthRole'), (route.target_id, route.resource_id).into(), 0_u8, array::ArrayTrait::span(@calldata));
```
issue is that there's already an implementation TIntoT in the corelib so both are applicable in the case of:
```
commands::set_entity(
      (route.target_id, route.resource_id).into(), (AuthRole { id: route.role_id })
        );
```
where route.target_id and route.resource_id are both u250 already

is there a way to specify which impl you want to use in this case ?